tests/sudorule/test_sudorule.yml

```diff
@@ -277,3 +376,26 @@
       state: absent
     register: result
     failed_when: result.changed
+
+  # cleanup
+  - name : Ensure sudocmdgroup is absent
+    ipasudocmdgroup:
+      ipaadmin_password: pass1234
+      name: test_sudorule
+      state: absent
+
+  - name: Ensure hostgroup is absent.
+    ipahostgroup:
+      ipaadmin_password: MyPassword123
+      name: cluster
+      host:
```

Comment:
The use of host with state:absent without action:member will result in an error. Please remove the host.